File: NTT/Models/Dto/UserDto.cs
using NTT.Domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace NTT.API.Models.Dto
{
    public class UserDto
    {

        public int Id { get; set; }
        public string Name { get; set; }
        
        public UserDto FromDomain(User user)
        {
            return new UserDto
            {

                Id = user.Id,
                Name = user.Name
            };

        }
        public User ToDomain()
        {
            return new User
            {

                Id = Id,
                Name = Name
            };

        }

    }

  
}

File: NTT.Frontend2/src/app/dto/userdto.ts
import { UserModel } from "../models/userModel";


export class UserDto {

  id : number;
  name: string;

    ToModel(): UserModel{
        var user = new UserModel();
        user.id = this.id;
        user.name=this.name

        return user;

    }

    FromModel(User : UserModel): UserDto{
        const Userdto = new UserDto();
            Userdto.id= User.id;
            Userdto.name = User.name;

        return Userdto;
    }


}
